Microsoft.HealthVault/Clients/Deserializers/ThingDeserializer.cs

```diff
@@ -144,5 +189,87 @@ public ThingDeserializer(IHealthVaultConnection connection)
 
             return result;
         }
+
+        public ThingBase DeserializeItem(XPathNavigator thingNav)
+        {
+            ThingBase result;
+            Guid typeId = new Guid(thingNav.SelectSingleNode("type-id").Value);
```

Comment:
the interface is internal .. so the class .. 

---
In reply to: [123106705](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/62#discussion_r123106705) [](ancestors = 123106705,123105225,123101596)